### PR TITLE
SQL diagram updates

### PR DESCRIPTION
**Pull request by ericharmeling**
Misc updates to the SQL diagram generator

Release justification: Non-production code changes

Release note: None

### CODE REVIEW

File: pkg/cmd/docgen/diagrams.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/cmd/docgen/extract"
 	"github.com/cockroachdb/cockroach/pkg/util/envutil"
-	"github.com/cockroachdb/errors"
```
Comment (mjibson) [inline pkg/cmd/docgen/diagrams.go:28]: revert this

**Review by madelynnblue — COMMENTED** [review]
I think you need to generate the BNF files and submit those with this PR too. `make` should do that.

**Comment by ericharmeling** [comment]
> I think you need to generate the BNF files and submit those with this PR too. `make` should do that.

Done.

**Review by madelynnblue — APPROVED** [review]
LGTM, but squash the commits before merging.

**Comment by ericharmeling** [comment]
Thanks @mjibson !

**Comment by knz** [comment]
The change is good but the CI tells you that the commit and PR description are missing a release justification string

```
[19:36:33]W:		 [Ensure commit message contains a release justification] Build Failed. No Release justification in the commit message or in the PR description.
[19:36:33]W:		 [Ensure commit message contains a release justification] Commits must have a Release justification of the form:
[19:36:33]W:		 [Ensure commit message contains a release justification] Release justification: <some description of why this commit is safe to add to the release branch.>
```

**Comment by ericharmeling** [comment]
@knz Updated both commit message and PR description. Thanks!

**Comment by otan** [comment]
hey @ericharmeling, you may need to backport this if you want this in 20.2

**Pull request merged**